src/design-system/inputs/EmailInput.tsx

```diff
@@ -2,37 +2,37 @@ import { useTranslation } from 'react-i18next'
 import TextInputGroup from './TextInputGroup'
 
 export default function EmailInput({
-	email,
-	setEmail,
-	errorEmail,
-	setErrorEmail,
-	...props
-}) {
-	const { t } = useTranslation()
+  email,
+  setEmail,
+  errorEmail,
+  setErrorEmail,
+  ...props
+}: any) {
```

Comment:
Bouhhh 👎  😄

Reply:
Wesh ça reste mieux que rien

Reply:
[not good enough](https://tenor.com/bbRZo.gif)

Reply:
bon je voulais mettre un gif mais c'est un fiasco ! 

normalement ça passe pas au build ça :)